Engine/Src/Ancona/System/Android/AndroidFileOperations.hpp

```diff
@@ -38,7 +42,8 @@ class AndroidFileOperations
     private:
         static AAssetManager * _assetManager;
         static std::string _internalPath;
-
+        
+        static bool IsFileInNonApkStorage(const std::string & filePath);
```

Comment:
You can put this in Android/FileOperations.cpp. Then you don't need to flag the class as friends.